Explorer/Models/DiveLogViewModel.swift

```diff
@@ -30,6 +31,14 @@ struct DiveLogViewModel {
         self.duration = "\(String(Int(diveLog.timeOut.timeIntervalSince(diveLog.timeIn) / 60))) min"
         self.userFullName = diveLog.user.fullName
         self.username = diveLog.user.username
+        
+        let mapUrl = URL(string: "https://maps.googleapis.com/maps/api/staticmap?center=\(diveLog.location.lat),\(diveLog.location.lon)&zoom=14&size=500x300&maptype=roadmap&markers=color:red%7Clabel:%7C\(diveLog.location.lat),\(diveLog.location.lon)")
```

Comment:
Breaking this up, or creating a helper method would be nice here. Something that breaks this URL up into several strings:
```
let baseUrl = "https://maps.googleapis.com/maps/api/"
let zoom = "14&size=500x300"
let center = "\(diveLog.location.lat),\(diveLog.location.lon)"`
let markers = "color:red%7Clabel:%7C\(diveLog.location.lat),\(diveLog.location.lon)"
let url = "\(baseUrl)staticmap?center=\(center)&zoom=\(zoom)&maptype=roadmap&markers=\(markers)"
```

A helper method could look like this:
```
func mapUrl(for location: Location) -> URL? {
    let baseUrl = "https://maps.googleapis.com/maps/api"
    let zoom = "14&size=500x300"
    let center = "\(location.lat),\(location.lon)"`
    let markers = "color:red%7Clabel:%7C\(location.lat),\(location.lon)"
    return "\(baseUrl)/staticmap?center=\(center)&zoom=\(zoom)&maptype=roadmap&markers=\(markers)"
}

let mapUrl = mapUrl(for: diveLog.location)
```